src/components/Card.js

```diff
@@ -1,62 +1,84 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleImageClick) {
-    this._name = name;
-    this._link = link;
+  constructor(
+    data,
+    cardSelector,
+    handleImageClick,
+    deleteCard,
+    likeCard,
+    unlikeCard
+  ) {
+    this.name = data.name;
+    this.link = data.link;
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._deleteCard = deleteCard;
+    this._likeCard = likeCard;
+    this._unlikeCard = unlikeCard;
```

Comment:
Please, delete `this._unlikeCard = unlikeCard;` from `Card` because you don't use it